### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jiwon-lieb**
When opening a pull request to submit a new KIP, please use the suggested template: https://github.com/klaytn/KIPs/blob/master/kip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your Github username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

**Comment by jiwon-lieb** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by baerae-colin — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# Branch name will be changed
+
+We will change the `master` branch to `main` at Nov 1, 2022. 
+After change branch policy, please check your local or forked repository settings.
```
Comment (smseo) [inline README.md]:
```suggestion
After the branch policy change, please check your local or forked repository settings.
```

File: README.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+# Branch name will be changed
+
+We will change the `master` branch to `main` at Nov 1, 2022.
```
Comment (jiwon-lieb) [inline README.md]:
```suggestion
We will change the `master` branch to `main` on Nov 1, 2022. 
```

**Review by smseo — APPROVED** [review]
LGTM. Thanks.

**Pull request merged**